packages/neuron-ui/src/containers/MainContent/state.ts

```diff
@@ -2,9 +2,14 @@ import { CapacityUnit } from '../../utils/const'
 
 export const initState = {
   tempWallet: {
```

Comment:
It seems that `tempWallet` have the same functionality as `createWallet` in the state.